source/Armament.h

```diff
@@ -42,6 +42,9 @@ class Visual;
 // distance away and velocity relative to the ship that is firing.
 class Armament {
 public:
+	Armament() = default;
+	Armament(const Armament &other);
```

Comment:
You'll need to implement the copy assignment operator as well, as well as move constructor and move assignment operator.